cmd/usecasevalidator/main.go

```diff
@@ -415,7 +414,8 @@ func validate(snapshot *pb.SnapshotWithType, info *useCaseInfo) (err error) {
 		}
 	}
 	if !isValid {
-		return fmt.Errorf("object '%s' is invalid: %w", id, err)
+		return fmt.Errorf("object '%s' (name: '%s') is invalid: %w",
+			id[len(id)-4:], pbtypes.GetString(snapshot.Snapshot.Data.Details, bundle.RelationKeyName.String()), err)
```

Comment:
Why we use 4 here?

Reply:
In most cases last 4 letters of object id are unique among all objects in particular space. 
I have shortened id in the output to make it more readable